### ISSUE TITLE
gRPC server missing some required information

### ISSUE THREAD

**Issue by xansec** [brittonhayes/warhammer#1]
This doesn't seem to run using the commands in the README.md.

```
$ docker-compose up
[+] Running 2/0
 ⠿ Container warhammer-grpc-1  Created                                                                                                                                                                      0.0s
 ⠿ Container warhammer-db-1    Created                                                                                                                                                                      0.0s
Attaching to warhammer-db-1, warhammer-grpc-1
warhammer-db-1    | 
warhammer-db-1    | PostgreSQL Database directory appears to contain a database; Skipping initialization
warhammer-db-1    | 
warhammer-grpc-1  | 10:17PM FTL No $CHANNEL_ID given. role=gRPC
warhammer-db-1    | 2022-01-26 22:17:46.738 UTC [1] LOG:  starting PostgreSQL 14.1 (Debian 14.1-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
warhammer-db-1    | 2022-01-26 22:17:46.738 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
warhammer-db-1    | 2022-01-26 22:17:46.738 UTC [1] LOG:  listening on IPv6 address "::", port 5432
warhammer-db-1    | 2022-01-26 22:17:46.740 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
warhammer-db-1    | 2022-01-26 22:17:46.791 UTC [26] LOG:  database system was shut down at 2022-01-25 01:13:45 UTC
warhammer-db-1    | 2022-01-26 22:17:46.800 UTC [1] LOG:  database system is ready to accept connections
warhammer-grpc-1 exited with code 1
```
What other info is required to get the server up and running?

**Comment by brittonhayes** [comment]
Hey @xansec! Thanks for opening this issue. Sorry it's taken me a few days to get back to you. Looks like I was initially intending to make this able to work as a Discord bot but... honestly... not super sure why I did that! So that `CHANNEL_ID` was it looking for a discord channel to use the bot with.

I've removed that functionality so it's a straight gRPC warhammer service now and it will spin up just fine with the commands in the README 😄 

When you're ready to try the fix, just run it with `docker-compose up --build` and it will build with that fresh image.
